utils/update_checkout.py

```diff
@@ -168,6 +168,11 @@ def update_single_repository(args):
                       "to rebase.\n")
 
             shell.run(["git", "submodule", "update", "--recursive"], echo=True)
+
+            # We also want to update all the tags with remote. By default git doesn't 
+            # do it and so we need to fetch it
+            # https://stackoverflow.com/questions/1204190/does-git-fetch-tags-include-git-fetch
```

Comment:
Can you refer to the git documentation instead of stack overflow. Also, are you sure you found all of the places we fetch?

Reply:
@gottesmm sure.

Reply:
BTW I have added older fetch request to include`--tags` which will avoid extra call as @jrose-apple suggested.